frontend/app/routes/_public+/apply+/$id+/review-information.tsx

```diff
@@ -231,30 +241,29 @@ export default function ReviewInformation() {
       <h2 className="mt-8 text-2xl font-semibold ">{t('apply:review-information.dental-title')}</h2>
       <dl>
         <DescriptionListItem term={t('apply:review-information.dental-private-title')}>
-          {dentalInsurance.private.length > 0
-            ? dentalInsurance.private.map((benefit) => {
-                return benefit;
-              })
-            : t('apply:review-information.dental-has-no-access')}
+          {dentalInsurance === 'yes' ? t('apply:review-information.dental-insurance-has-access') + dentalInsurance : t('apply:review-information.dental-no-access')}
```

Comment:
Move the `yes` string into the translation string. Otherwise, you could end up rendering:

```
A accès à une assurance ou une couverture dentaire : yes
```

Also if the value of `dentalInsurance` changes for some reason, then the page won't render as expected.